### DISCUSSION TITLE
No Audio in the RTSP stream

### DISCUSSION THREAD

**Discussion by TGda** [mrlt8/docker-wyze-bridge#75] [Q&A]
Its just me or there is no audiro on the RTSP?  My config:
Synology 220+
Docker
Surveillance Station
Wyze V3
codec h.264

PD.  Wyze V2 are connected directly without the bridge and audio is working.

Thanks in advance.

**Comment by mrlt8** [comment]
No audio at the moment, just video. But it is something I would want to take a look into eventually. If anyone else wants to have a look and help out, the tutk library does have [audio support](http://www.sunipcam.com/sdk/UserManual/APIRefGuide/_a_v_a_p_is_8h.html#a2851fe99900f17c34d6f9bfc1492e014).
